src/main/java/uk/gov/hmcts/cmc/claimstore/services/notifications/NotificationReferenceBuilder.java

```diff
@@ -13,103 +13,103 @@ private NotificationReferenceBuilder() {
     }
 
     private static String reference(
-        final String template,
-        final String toWhom,
-        final String claimReferenceNumber
+        String template,
+        String toWhom,
+        String claimReferenceNumber
     ) {
         return String.format(template, toWhom, claimReferenceNumber);
     }
 
     public static class MoreTimeRequested {
 
-        public static final String TEMPLATE = "more-time-requested-notification-to-%s-%s";
+        public static String TEMPLATE = "more-time-requested-notification-to-%s-%s";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "TEMPLATE" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS3008)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this "public static TEMPLATE" field final [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS1444)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make TEMPLATE a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)